ra/ra_test.go

```diff
@@ -981,6 +981,7 @@ func TestPerformValidationAlreadyValid(t *testing.T) {
 	// Create a finalized authorization
 	exp := ra.clk.Now().Add(365 * 24 * time.Hour)
 	authz := core.Authorization{
+		ID:             "1337",
```

Comment:
Here and below: rather than building a `core.Authorization` and calling `bgrpc.AuthzToPB` on it (line 997), simply build the `corepb.Authorization` directly. This will allow us to more readily get rid of the conversion functions.